packages/lab/src/calendar/internal/CalendarNavigation.tsx

```diff
@@ -171,33 +172,27 @@ export const CalendarNavigation = forwardRef<
     moveToNextMonth(event);
   };
 
-  const handleMonthSelect: SelectionChangeHandler<DropdownItem> = (
-    event,
-    month
-  ) => {
+  const handleMonthSelect = (event: SyntheticEvent, month: DropdownItem[]) => {
     if (month) {
```

Comment:
month will always be true because it's an array. You'd have to check if month[0] is defined.